### ISSUE TITLE
[BUG] Blank Screen on the Map Editor

### ISSUE THREAD

**Issue by JasminDreasond** [Durtur/Dungeoneer#99]
**System information**
Linux Ubuntu 23
Game 1.2.5 

**Describe the bug**
The Map Editor simply opens a blank window. At first everything seems to behave normally, but at some random moment when I try to open the map editor, this will happen, and not even clearing the software cache will not solve the problem.

**Expected behavior**
The Map Editor opens normally.

**Screenshots**
It's simply a blank page, I don't see the need to take a screenshot of a 100% solid color screen.

**Comment by Durtur** [comment]
If I understand you correctly the map window opens normally but at some random point you are not able to open it? Is the window then completely unresponsive or are you able to open the developer tools with ctrl+shift+i ?

**Comment by JasminDreasond** [comment]
yep. the window becomes completely unusable, but the page continues to work. I updated to version 1.2.6, but the problem continues.

the first time I had suddenly managed to stop the bug somehow, but now I can no longer in any way remove the bug because I don't know what to do. I would recommend you first update electron to last electron version, this way maybe I can also try to figure out what is going on.

**Comment by Durtur** [comment]
Last time I built this for linux it was the sharp image library (https://sharp.pixelplumbing.com/) that was causing the window to become completely unresponsive on Linux, I guess the solution might be to move it into the main process (it should be there anyway, along with all nodejs stuff). Updating electron might help yes, unfortunately I don't have a lot of time to work on this project these days but I will take a look at this

**Comment by Durtur** [comment]
Upon further reading it seems like the sharp library needs to be installed on a machine that has the same runtime architecture. So if you would like to fix the build for your machine you can try to remove it and add it again, e.g.

**yarn remove sharp
yarn add sharp**

**Comment by JasminDreasond** [comment]
yep. When I did this, many things that were previously working incorrectly were suddenly working correctly. I would recommend you put this information in "README.md".

**Comment by JasminDreasond** [comment]
I'm still a little confused. Why doesn't saving the map save the position of tokens and players?
I tried to find some documentation on how to use the application, but I couldn't find it.

**Comment by Durtur** [comment]
The map doesn't save the state of the tokens. Instead I would recommend you add your players in the player menu and set their tokens, so they are added on map load. As for monsters, you can add tokens to the individual statblocks and add them to the map by doing ctrl+shift+e and then add to map. That way you track the combat state and tie each row to a specific token on the map

**Comment by Durtur** [comment]
Unfortunately I have not set up any sort of documentation but there is a little bit outdated usage guide in the about window, (?) icon
